<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberFormatDialog</class>
 <widget class="QDialog" name="NumberFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Формат числа</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Формат</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="auto_radioButton">
        <property name="text">
         <string>Авто</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sci_radioButton_2">
        <property name="text">
         <string>Экспоненциальный</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="regular_radioButton_3">
        <property name="text">
         <string>Десятичная дробь</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Число десятичных знаков:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="degSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="meaningfulDegCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Отсчитывать значащие разряды</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>regular_radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>degSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regular_radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>meaningfulDegCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regular_radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
